kubearmor-grafana-loki/logs/tutorial_grafana.md

```diff
@@ -0,0 +1,19 @@
+### About
+This tutorial shows how to export kubearmor telemetry data to grafana using the kubearmor receiver and [loki exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/lokiexporter)
+
+### Steps
+1. Follow [this tutorial](https://github.com/kubearmor/KubeArmor/blob/ce18fee4f87be7786dc1275aeb94ab7096c8b590/getting-started/deployment_guide.md#L20-L19) to set up Kubearmor
+2. Follow this [tutorial](https://github.com/Chinwendu20/OTel-receiver/blob/otel/example/tutorial.md#create-a-custom-opentelemetry-collector-distribution) to create an opentelemetry custom collector.
```

Comment:
Please do not use personal repos.

Reply:
Thank you. Here just that the tutorial file is currently in my personal repo as my PR on the receiver has not yet been merged in kubearmor's org